.github/workflows/ci.yaml

```diff
@@ -0,0 +1,57 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  server-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12]
+    services:
+      postgres:
```

Comment:
Postgres is an interesting choice :) Do you think Spanner, Bigtable, or Firestore are viable here?

Reply:
I think Firestore would be good to explore using.